### PR TITLE
feat(ssh): Config ssh key permanently and globally

### PR DESCRIPTION
**Pull request by VoVAllen**
Change:
- When use specify a new key in `envd bootstrap`, the old key will be copied to a new one with a random name. And the entries in ssh config file is changed accordingly
- The ssh flag in up and build is kept but turned to hidden

### CODE REVIEW

File: pkg/app/bootstrap.go [hunk 1]
```diff
@@ -42,13 +50,81 @@ var CommandBootstrap = &cli.Command{
...
 }
 
 func bootstrap(clicontext *cli.Context) error {
+	// If not specified, generate only if key doesn't exist
+	sshKeyPair := clicontext.StringSlice("ssh-keypair")
+	if len(sshKeyPair) != 0 && len(sshKeyPair) != 2 {
+		return errors.Errorf("Invliad ssh-keypair flag")
+	} else if len(sshKeyPair) == 0 {
+
+		keyExists, err := sshconfig.DefaultKeyExists()
```
Comment (gaocegege) [inline pkg/app/bootstrap.go:70]: Is it possible, or easy, to show the default key paths in CLI flag?

File: pkg/app/bootstrap.go [hunk 2]
```diff
@@ -78,3 +154,7 @@ func bootstrap(clicontext *cli.Context) error {
 	}
 	return nil
 }
+
+func FileExists() {
```
Comment (gaocegege) [inline pkg/app/bootstrap.go:158]: I am wondering if we can remove it.

File: pkg/util/fileutil/namegenerator.go [hunk 3]
```diff
@@ -0,0 +1,869 @@
+// Copyright 2022 The moby Authors
```
Comment (gaocegege) [inline pkg/util/fileutil/namegenerator.go:1]: This should be the second line. Keep envd the first line.

File: pkg/app/bootstrap.go (hunk 1 above)
Comment (VoVAllen) [reply pkg/app/bootstrap.go:71]: Added

**Review by gaocegege — APPROVED** [review]
LGTM :+1:
/lgtm

**Pull request merged**